src/styles/search-panel.scss

```diff
@@ -702,6 +702,10 @@
 
 	&__list {
 		width: 100%;
+		height: 300px;
```

Comment:
We have to save previous behavior. Result list should take up all available space. Probably you need to use `flex-grow`